common/enums/available-languages/index.ts

```diff
@@ -0,0 +1,29 @@
+export enum AvailableLanguages {
```

Comment:
Please don't put anything to `/common`

Define it on the server and import via `api` on the client

Reply:
hah, I did that at first and then switched to common, thought it should be there

Reply:
fixed